### DISCUSSION TITLE
[Enhancement] [date-picker]  时间选择器datepicker 第二次选择相同时期，change事件不会触发

### DISCUSSION THREAD

**Discussion by HouZhiGuoSir** [element-plus/element-plus#17433] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `date-picker`

## Description

当我选择了一个日期范围后，再选择相同的日期范围，change事件不会触发，导致客户以为数据有问题，刷新了一下页面才可以

### Additional comments

_(empty)_

**Comment by nasirovx** [comment]
Yes, this enhancement is important. When the user selects the same date range again, the change event should be triggered to avoid confusion and ensure that the data is correctly updated without needing to refresh the page.
